test/e2e/search_test.go

```diff
@@ -98,6 +99,21 @@ registries = ['{{.Host}}:{{.Port}}']`
 		Expect(search.LineInOutputContains("quay.io/libpod/gate")).To(BeTrue())
 	})
 
+	It("podman search image with description", func() {
+		search := podmanTest.Podman([]string{"search", "quay.io/libpod/whalesay"})
+		search.WaitWithDefaultTimeout()
+		Expect(search.ExitCode()).To(Equal(0))
+		lines := search.OutputToStringArray()
+		match := false
+		for _, v := range lines {
+			match, _ = regexp.MatchString(`^quay.io\s+quay.io/libpod/whalesay\s+Static image used for automated testing.+$`, v)
+			if match {
+				break
+			}
+		}
+		Expect(match).To(BeTrue())
```

Comment:
This is failing in CI

Reply:
Looks like the original issue #7131 is lacking a fix for the remote client. I thought I checked that, hm...